Document/0x05b-Basic-Security_Testing.md

```diff
@@ -66,6 +66,62 @@ Once you've configured the network and established a connection between the test
 
+As mentioned before, starting at Android 7, Android device will no longer trust user CA certificates by default, unless specified in the application. In the following section, we explain two methods to bypass this android security mesure.
+
+#### Bypassing the Network Security Configuration
+
+From Android 7 onward, the network security configuration allows apps to customize their network security settings, by defining which CA certificates the app will be trusting. The CA certificates trusted by the app can be a system trusted CA as well as a user non-public CA.
+Network security configuration uses an XML file where the app specify which CAs certificates will be trusting.
+
+res/xml/network_security_config.xml:
+```shell
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <base-config>
+        <trust-anchors>
+            <certificates src="@raw/extracas"/>
+            <certificates src="system"/>
+        </trust-anchors>
+    </base-config>
+</network-security-config>
+```
+The apps must include an entry in the manifest file to point to the network security configuration file.
+```shell
+<?xml version="1.0" encoding="utf-8"?>
+<manifest ... >
+    <application android:networkSecurityConfig="@xml/network_security_config"
+                    ... >
+        ...
+    </application>decomp
+</manifest>
+```
+
+In order to intercept an application's traffic which Android version is 7 and above, you must follow the steps below:
+
+- Decompile the app using decompiling tools.[Manual static Analysis] provides details about decompiling (https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05b-Basic-Security_Testing.md#manual-static-analysis)
+- Make the application trust the proxy's certificate by creating a network security configuration with the giving certificate as explained above
+- Repackage the app.[Android developer documentation](https://developer.android.com/studio/publish/app-signing#signing-manually) explains how it's done. 
+
+Note that even if this method is quite simple its major drawback is that you have to apply this operation for each application you want to evaluate which is redundant and annoying.
+
+##### Adding the Proxy's certificate among system trusted CAs
+
+In order to avoid the obligation of configuring the Network Security Configuration for each application, we must force the Device to accept the proxy's certificate as one of the system strusted certificates, to do so follow the steps below:
+
+- Making the system files writable which requires rooting the Device.Find instructions on how to [root](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05b-Basic-Security_Testing.md#connecting-to-an-android-virtual-device-avd-as-root) your device later in this chapter.Run 'mount' command to make sure the /system is writable, if it still not the case run the following command 'mount -o rw,remount -t ext4 /system'
```

Comment:
Device => device 
Space after "chapter."
Run the 'mount' command to make sure that /system is writable, if this is still not the case run the following command 'mount -o rw,remount -t ext4 /system'

Reply:
already done. please base your request on the latest commits

Reply:
Please change "the /system" to "that /system". That's missing, then we can merge :-)